src/Symfony/Component/Asset/Tests/fixtures/manifest-invalid-json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
It's clever to not use `.json` extension to avoid fabbot issues ... but that could complicate the maintenance of this. Imagine looking at this conde in 1 year and wondering: why this file doesn't have an extension? Is this an error? Should I add the extension now? etc.

So, let's add the `.json` extension and forget about fabbot issues.